packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4012,7 +4030,16 @@ export class EcdsaPkRecover extends Op {
 		super();
 		this.line = line;
 		assertLen(args.length, 1, line);
-		this.curveIndex = Number(args[0]);
+		const argument = args[0];
+		if (argument == "0" || argument == CurveTypeArgument.secp256k1) this.curveIndex = 0;
```

Comment:
`argument == "0"` what does it mean?

Reply:
to check the if the arugment pass in is in number 0 or not

Reply:
but let me make the same as the other opcode